drivers/session_mock.go

```diff
@@ -2,17 +2,33 @@ package drivers
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	"github.com/livepeer/go-livepeer/net"
 	"github.com/stretchr/testify/mock"
 )
 
 type MockOSSession struct {
 	mock.Mock
+	waitForCh bool
+	waitCh    chan interface{}
```

Comment:
Changed, but what's the reason for this change? I'm usually using `interface` because I don't need to create value to send into it - I can just send `nil`.